lib/mercadopago/collection.rb

```diff
@@ -101,10 +102,39 @@ def self.notification_preapproval(access_token, preapproval_id)
     #     subscription_payment::
     #       Subscription fee.
     #
-    def self.search(access_token, search_hash = {}, sandbox = false)
+    def self.search(access_token, search_hash = {}, sandbox = false, auto_paginate = false)
```

Comment:
@cleicar could you update the documentation in this file on lines 39-41 to address the new `auto_paginate` param? Just a brief explanation of what it does. 

We'll also need an explanation in the README, so that developers know that the option exists without diving into the code.

Reply:
@cleicar Did you get a chance to review this comment?

Reply:
@cleicar Did you get a chance to see this?